src/main/java/io/vertx/core/shareddata/AsyncMap.java

```diff
@@ -81,15 +82,15 @@
    * @param ttl  The time to live (in ms) for the entry
    * @param completionHandler  the handler
    */
-  void putIfAbsent(K k, V v, long ttl, Handler<AsyncResult<V>> completionHandler);
+  void putIfAbsent(K k, V v, long ttl, Handler<AsyncResult<@Nullable V>> completionHandler);
```

Comment:
the second param `V` should also be `@Nullable`

Reply:
No, null values are forbidden in `AsyncMap`.

Reply:
ah ok makese sense then, perhaps we should say it in the javadoc as well then